package.json

```diff
@@ -110,6 +110,7 @@
 		"@testing-library/jest-dom": "5.16.1",
 		"@testing-library/react": "11.2.2",
 		"@testing-library/react-native": "9.0.0",
+		"@testing-library/user-event": "^14.0.0-beta.13",
```

Comment:
This the latest available version, and is the recommended one over the previously stable version (`13.5.0`)

From the [official docs](https://testing-library.com/docs/user-event/intro):

> These docs describe `user-event@14.0.0-beta`. This is still a pre-release and might be subject to breaking changes if they should be deemed necessary in the light of feedback we receive for this version. All planned breaking changes are already implemented though.
If you are starting or actively working on a project, we recommend to use this pre-release, as it includes important bug fixes and new features.